src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -56,41 +62,62 @@
       {% if categoryForm.id_parent is defined %}
       <div class="form-group row">
         <label class="form-control-label">
-          {{ 'Parent category'|trans }}
+          {{ 'Parent category'|trans({}, 'Admin.Catalog.Feature') }}
         </label>
         <div class="col-sm">
           {{ form_widget(categoryForm.id_parent) }}
         </div>
       </div>
       {% endif %}
 
+      {% if categoryForm.description is defined %}
       <div class="form-group row">
-        {{ ps.label_with_help(('Description'|trans({}, 'Admin.Global')), ('Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}')) }}
+        <label class="form-control-label">
+          {{ 'Description'|trans({}, 'Admin.Global') }}
+        </label>
         <div class="col-sm">
           {{ form_widget(categoryForm.description) }}
+          <small class="form-text">
+            {{ 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}' }}
+          </small>
         </div>
       </div>
+      {% endif %}
 
       <div class="form-group row">
-        {{ ps.label_with_help(('Category Cover Image'|trans), ('This is the main image for your category, displayed in the category page. The category description will overlap this image and appear in its top-left corner.'|trans({}, 'Admin.Catalog.Help'))) }}
+        <label class="form-control-label">
+          {{ 'Category Cover Image'|trans({}, 'Admin.Catalog.Feature') }}
```

Comment:
i just wanted to keep exactly same translation key, don't know if we would lose translation if we change letters. should I change it?

Reply:
We will lose the actual translation, yes, but I don't think it is going to be a problem since it will be translated again - even more quickly, the translation already exists, it is just about "the look" of the wording. ;-)

Reply:
done.